addons/zeus/CfgVehicles.hpp

```diff
@@ -98,14 +98,14 @@ class CfgVehicles {
         category = QGVAR(Repair);
         displayName = CSTRING(ModuleAddSpareTrack_DisplayName);
         function = QFUNC(moduleAddSpareTrack);
-        icon = QPATHTOF(UI\Icon_Module_Zeus_Medic_ca.paa);//@todo
```

Comment:
Remove that icon as well, or is it used elsewhere?

Reply:
It is used in actual medical modules like `moduleSetMedic`. Though ace's medic cross could be replaced with files from ui_f\data too. I'll handle other outdated icons in a separate PR.